.github/workflows/pr.yml

```diff
@@ -54,6 +54,7 @@ jobs:
         run: |
           rm -r docs
           /tmp/tfplugindocs
+          git diff
```

Comment:
this helps identify the doc diff for those of us who have trouble running it locally